persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/StructuralVariantMapper.xml

```diff
@@ -51,6 +51,49 @@
         structural_variant.DRIVER_TIERS_FILTER_ANNOTATION AS "${prefix}driverTiersFilterAnn"
     </sql>
 
+    <sql id="whereInMultipleMolecularProfiles">
+        <where>
+            <if test="sampleIds != null and !sampleIds.isEmpty()">
```

Comment:
Any idea what to expect of the runtime performance of this query?

Reply:
No idea, but it's similar to what's in mutation query and even at other places